index.html

```diff
@@ -4,32 +4,40 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheets.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
Remove this link

Reply:
Only remove line 8   <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">

Reply:
so, looking at w3schools, they do not teach you what to put in the stylesheet, only what you put in the actual index page, do you know of where else i could find how to update the style page correctly? when i remove that link, the box and some other parts disappear...

Reply:
> 

I don't quite understand your question, but I think you might be asking about inline CSS. You can apply styling directly within the index.html to an element, however, this is not best practice. You can look up inline css tutorials online. I asked you to  remove line 8 ( <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">) because that link is not valid CSS. Also, your styles might have disappeared because there was a div wrapping all your content. If this doesn't make sense, I can explain next time we're on screenshare. So, the best way to style any element on your index.html or any page would be by using CSS selectors like classes, IDs and HTML elements. This will naturally make more sense as you work on projects, so don't worry if you don't understand some things yet.

Reply:
ok. I think I understand. Thanks!

Reply:
so, once I have made the changes, git added, commited, and pulled, what should I do next? Should I Merge the pull request or create a new pull request?